code/common/os.q

```diff
@@ -8,9 +8,9 @@ del:{system("rm ";"del ")[NT],pth x}
 deldir:{system("rm -r ";"rd /s /q ")[NT],pth x}
 hdeldir:{[dirpath]
  diR:{$[11h=type d:key x;raze x,.z.s each` sv/:x,/:d;d]};
- nuke:hdel each desc diR@;
- nuke hsym`$dirpath;
- .lg.o[`deldir;"deleting from  directory"]}
+ nuke:hdel each desc 1_diR@;
+ .lg.o[`deldir;"deleting from  directory: ",raze string dirpath];
```

Comment:
isn't dirpath already a string? So you are stringing a string which is why you are needing to raze?

Reply:
Since you have a type check above you no longer need `raze string` here. You can assume it's a string at this point.
```suggestion
 .lg.o[`deldir;"deleting from  directory: ",dirpath];
```

Reply:
yes sorry that slipped me by meant to remove that thanks